ddtrace/internal/core/__init__.py

```diff
@@ -276,56 +282,68 @@ def __getattr__(name):
     raise AttributeError
 
 
+_CURRENT_CONTEXT = contextvars.ContextVar("ExecutionContext_var", default=ExecutionContext(ROOT_CONTEXT_ID))
+_CONTEXT_CLASS = ExecutionContext
+
+
 def _reset_context():
+    """private function to reset the context. Only used in testing"""
     global _CURRENT_CONTEXT
     _CURRENT_CONTEXT = contextvars.ContextVar("ExecutionContext_var", default=ExecutionContext(ROOT_CONTEXT_ID))
 
 
-_reset_context()
-_CONTEXT_CLASS = ExecutionContext
-
-
 def context_with_data(identifier, parent=None, **kwargs):
     return _CONTEXT_CLASS.context_with_data(identifier, parent=(parent or _CURRENT_CONTEXT.get()), **kwargs)
 
 
-def get_item(data_key, span=None):
-    # type: (str, Optional[Span]) -> Optional[Any]
+def get_root() -> ExecutionContext:
```

Comment:
This duplicates the functionality of this module's `__getattr__` function.

Reply:
Oh yes, you can access the root through core.root indeed.
Removed.